lib/sarif/base_sarif.rb

```diff
@@ -25,12 +25,21 @@ def initialize(scan_report, config = {}, repo_path = nil)
       @issues = Set.new
       @config = config
       @repo_path = repo_path || Dir.getwd # Fallback, we should make repo_path required
+      @scanner_config = scanner_config
     end
 
     def base_path
       @base_path ||= @repo_path.nil? ? nil : File.expand_path(@repo_path)
     end
 
+    def check_warn
+      unless @scanner_config.nil?
+        @scanner_config.dig(:scanner_configs, @scan_report.scanner_name, "warn_message") || false
+      end
+
+      false
```

Comment:
Doesn't this method always return `false`?

Reply:
Fixed, thanks for catching that